### PR TITLE
update network details

### PR DESCRIPTION
**Pull request by mason5991**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,27 +3,71 @@
...
+
+- Payment amount: 0.01 LINK
+- Oracle address: 0xA96474C1A08374EFd0F3C9BC7153FDA7A6c8d9e1
+- LINK token address: 0xb0897686c545045afc77cf20ec7a532e3120e0f1
+- Job ID(Truflation): b04c2a85143c43089c1befe7c41dea93
+
+**Important**
+**The LINK provided by the [Polygon (Matic) Bridge](https://wallet.polygon.technology/bridge) is not ERC-677 compatible, so cannot be used with Chainlink oracles. However, it can be [converted to the official LINK token on Polygon (Matic) using Chainlink's PegSwap service](https://pegswap.chain.link/?_ga=2.130074441.756683581.1650345653-1161789045.1649048909)**
+
+### Mumbai Testnet (Chain ID: 80001)
```
Comment (JGJP) [inline README.md]: Should we specify that this is Polygon's testnet?

Comment (mason5991) [reply README.md]: No. Only for polygon mainnet. They don't need to do in testnet.

Comment (JGJP) [reply README.md]: I mean it's maybe clearer if we write `Mumbai (Polygon) Testnet (Chain ID: 80001)` no?

Comment (mason5991) [reply README.md]: Oh I see. Renamed to Mumbai (Polygon) Testnet

**Pull request merged**